### PR TITLE
DOCS-10222 updating info about deprecated port.

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

**Review by sarjeet2013 — DISMISSED** [review]

**Comment by sundippatel** [comment]
Would we need to update the Port Configuration section on the install-self-managed.adoc page?
It lists port 5044 for Anypoint Monitoring and Visualizer.

Same question for the Hostname Configuration section that follows on the same page.

File: modules/ROOT/pages/install-prereqs.adoc [hunk 1]
```diff
@@ -189,19 +194,19 @@ Because the following endpoints use mutual TLS authentication to establish the c
 ** `configuration-resolver.prod-eu.msap.io`
 
 [%header%autowidth.spread]
-.Port IP Addresses and Hostnames to Whitelist
+.Port IP Addresses and Hostnames to add to the allow list
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/install-prereqs.adoc]: capitalize as 'Add to the Allow List'?

**Review by wwarner-salesforce — DISMISSED** [review]

**Comment by sundippatel** [comment]
We also mention port 5044 in the manage-monitor-applications.adoc page. Could we add the deprecation notice there also (under the Anypoint Monitoring section)?

**Review by sundippatel — CHANGES_REQUESTED** [review]
Left some comments on a couple of pages were we mention port 5044 without calling out it's deprecation.

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 2]
```diff
@@ -15,11 +15,16 @@ By default, Runtime Fabric is configured to send application metrics to Anypoint
...
 
-To verify outbound network connectivity, ensure that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 5044 to:
+To verify outbound network connectivity, ensure that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 443 to:
 
 . US control plane: `dias-ingestor-nginx.prod.cloudhub.io`
 . EU control plane: `dias-ingestor-nginx.prod-eu.msap.io`
 
+[NOTE]
+====
+If you are using a version of Anypoint Runtime Fabric older than 1.8.50, uses port 5044 instead of port 443
```
Comment (sundippatel) [inline modules/ROOT/pages/manage-monitor-applications.adoc]: "uses" -> "use"

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 3]
```diff
@@ -15,11 +15,16 @@ By default, Runtime Fabric is configured to send application metrics to Anypoint
 
 If you need an outbound proxy, configure Runtime Fabric to use a SOCKS5 proxy by following the instructions in xref:manage-proxy.adoc[Manage Proxies Used by Runtime Fabric].
 
-To verify outbound network connectivity, ensure that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 5044 to:
+To verify outbound network connectivity, ensure that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 443 to:
 
 . US control plane: `dias-ingestor-nginx.prod.cloudhub.io`
```
Comment (sundippatel) [inline modules/ROOT/pages/manage-monitor-applications.adoc]: Should update to "dias-ingestor-router.us-east-1.prod.cloudhub.io" for agent version 1.8.50 and later

File: modules/ROOT/pages/install-prereqs.adoc [hunk 4]
```diff
@@ -189,19 +194,19 @@ Because the following endpoints use mutual TLS authentication to establish the c
...
 |===
 | Port | Protocol | Hostnames | Description
 .2+| 443 .2+| AMQP over WebSockets | US control plane: `transport-layer.prod.cloudhub.io` | Runtime Fabric message broker for interaction with the control plane.
 | EU control plane: `transport-layer.prod-eu.msap.io` | Runtime Fabric message broker for interaction with the control plane.
+.2+| 443 (v1.8.50, or later) .2+| TCP (Lumberjack) | US control plane: `dias-ingestor-router.us-east-1.prod.cloudhub.io` | Anypoint Monitoring agent for Runtime Fabric.
+| EU control plane: `dias-ingestor-router.us-east-1.prod.cloudhub.io` | Anypoint Monitoring agent for Runtime Fabric.
```
Comment (sundippatel) [inline modules/ROOT/pages/install-prereqs.adoc]: The EU control plane hostname is the same as the US control plane.. I think that might be an error.
cc @sarjeet2013 to confirm.

File: modules/ROOT/pages/manage-monitor-applications.adoc (hunk 3 above)
Comment (sundippatel) [reply modules/ROOT/pages/manage-monitor-applications.adoc]: Same with below (if necessary).

File: modules/ROOT/pages/install-prereqs.adoc (hunk 4 above)
Comment (sarjeet2013) [reply modules/ROOT/pages/install-prereqs.adoc]: yes, it's an error. EU endpoint is: `dias-ingestor-router.eu-central-1.prod-eu.msap.io`

File: modules/ROOT/pages/install-prereqs.adoc (hunk 1 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-prereqs.adoc]: Removing table title.

File: modules/ROOT/pages/install-prereqs.adoc (hunk 4 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-prereqs.adoc]: Updated.

File: modules/ROOT/pages/manage-monitor-applications.adoc (hunk 3 above)
Comment (sleslie23) [reply modules/ROOT/pages/manage-monitor-applications.adoc]: Updated hostnames.

File: modules/ROOT/pages/manage-monitor-applications.adoc (hunk 2 above)
Comment (sleslie23) [reply modules/ROOT/pages/manage-monitor-applications.adoc]: Fixed

File: modules/ROOT/pages/install-prereqs.adoc [hunk 5]
```diff
@@ -149,7 +149,12 @@ The following table lists all ports that must be open, along with the respective
 | 443 | TCP | HTTPS | Internet | Controller nodes | Allow inbound requests to Mules
 | 443 | TCP | AMQP over WebSockets | Controller nodes | Internet | Anypoint Platform management services
 | 443 | TCP | HTTPS | All nodes | Internet | API Manager policy updates, API Analytics Ingestion, and Resource retrieval (application files, container images).
+| 443 (v1.8.50, or later) | TCP | Lumberjack | All nodes | Internet | Anypoint Monitoring, Anypoint Visualizer
 | 5044 | TCP | Lumberjack | All nodes | Internet | Anypoint Monitoring, Anypoint Visualizer
```
Comment (sarjeet2013) [inline modules/ROOT/pages/install-prereqs.adoc]: Can you add `deprecated` to 5044 as well? Similar to how we added to https://docs.mulesoft.com/runtime-fabric/1.8/install-prereqs#port-ip-addresses-and-hostnames-to-whitelist

Comment (sleslie23) [reply modules/ROOT/pages/install-prereqs.adoc]: Added (deprecated)

**Review by sarjeet2013 — APPROVED** [review]

**Review by sundippatel — APPROVED** [review]

**Review by Mulesoft-Mysti — APPROVED** [review]

**Pull request merged**